operations/commit_queue.go

```diff
@@ -690,7 +698,8 @@ func (p *moduleParams) addModule(ac *legacyClient, rc *legacyClient) error {
 
 func showCQMessageForProject(ac *legacyClient, projectID string) {
 	projectRef, _ := ac.GetProjectRef(projectID)
-	if projectRef != nil && projectRef.CommitQueue.Message != "" {
+	err := projectRef.MergeWithParserProject("")
+	if err != nil && projectRef != nil && projectRef.CommitQueue.Message != "" {
```

Comment:
You can't combine these checks; MergeWithParserProject will (and rightly should) panic if projectRef is nil